app.js

```diff
@@ -10,10 +11,16 @@ app.get("/:user/:repo/compile", (req, res) => {
   const user = req.params.user;
   const repo = req.params.repo;
 
-  util.getReadme(user, repo).then(() => {
-    util.serveDocs(app, user, repo);
-    res.send(`Successfully Compiled Docs! View them at /${user}/${repo}`);
-  });
+  try {
+    util.compileDocs(user, repo).then(() => {
+      util.serveDocs(app, user, repo);
+      res.send(`Successfully Compiled Docs! View them at /${user}/${repo}`);
+    });
+  } catch (error) {
+    res.send(
+      `Oops. We couldn't render these docs. Check your config if you have one and try again!`
+    );
+  }
```

Comment:
Does `compileDocs` have an error handler? That would be better than `try / catch`